data/src/main/kotlin/com/alfresco/content/data/AnalyticsManager.kt

```diff
@@ -38,7 +58,7 @@ class AnalyticsManager(val session: Session = SessionManager.requireSession) {
      */
     fun taskFiltersEvent(name: String) {
         val params = repository.defaultParams()
-        val eventName = "event_${name.replace(" ","_")}"
+        val eventName = "event_${name.replace(" ", "_")}"
```

Comment:
are we changing the event name?

Reply:
we are not changing the event name. It just added the space over there on refactoring